index.bs

```diff
@@ -1003,6 +1007,14 @@ spec: mimesniff; urlPrefix: https://mimesniff.spec.whatwg.org/#
               parameters may be present depending on the codec.
             </p>
           </li>
+          <li>
```

Comment:
Sorry, just coming back to this, how is the actual validity checks performed? What exactly is being checked. 

It can remain hand-wavy, but maybe we should have a note that one might use a lookup table or something? 

I guess it depends on if we are going to hit interop issue if this is not precisely specified, or will this be ok?

Reply:
I'm a little out of my depth as to the algorithm.  I would need to defer to a codec experts how the enum values map onto codec metadata like that defined [here](https://github.com/webmproject/vp9-dash/blob/main/VPCodecISOMediaFileFormatBinding.md#codecs-parameter-string).

Moreover, unclear how this specification could define behavior given the open-ended nature of current and future mime types that may be used with it.